django/http/request.py

```diff
@@ -711,8 +711,8 @@ def __repr__(self):
         return "<%s: %s>" % (self.__class__.__qualname__, self)
 
     @cached_property
-    def params(self):
-        params = self._params.copy()
+    def range_params(self):
```

Comment:
Question: What's the thinking around calling this "range" parameters? The RFC doesn't really give them a name, but the closest might be "type parameters", since they're the parameters for the type itself.

Reply:
> Spoke to Jake via Discord, they do not feel strong about this naming.

I am Jake and I approve this message :heavy_check_mark: